### DISCUSSION TITLE
v2.0.0

### DISCUSSION THREAD

**Discussion by Minemetero** [lingbopro/easy-github-hosts#22] [Ideas]
是时候开始重启这个项目了
我们要进行飞跃性的，史诗级别的升级

**Comment by lingbopro** [comment]
哦？

**Reply by Minemetero** [reply]
你给我钱我到日本搭个服务器做VPN :)

**Reply by lingbopro** [reply]
没钱:(

**Reply by lingbopro** [reply]
我觉得VPN已经超出这个项目的范围了

**Comment by Minemetero** [comment]
单独靠Hosts已经不能满足需求啦 我们在搞点技术增强一下

**Reply by lingbopro** [reply]
emmm...先在dev搞罢

**Reply by Minemetero** [reply]
https://github.com/docmirror/dev-sidecar
我们去偷点技术来

**Reply by lingbopro** [reply]
> https://github.com/docmirror/dev-sidecar 我们去偷点技术来

这个项目是Electron啊，我擅长

**Reply by Minemetero** [reply]
> > https://github.com/docmirror/dev-sidecar 我们去偷点技术来
> 
> 这个项目是Electron啊，我擅长

我一窍不通

**Reply by Minemetero** [reply]
要不这样 2.0我们增强功能，3.0搞个UI (主要是我UI不会，先把技术的搞了=) )

**Reply by lingbopro** [reply]
UI的话完全可以开新项目了

**Reply by Minemetero** [reply]
是的

**Reply by lingbopro** [reply]
新项目的话肯定就不止加速了，可以塞点新功能进去

**Reply by Minemetero** [reply]
lol塞个计算器

**Reply by lingbopro** [reply]
emmm...这就不必了吧...

**Comment by Minemetero** [comment]
靠内网穿透你看行不行

**Reply by lingbopro** [reply]
这种要服务器，与其自己搭还不如用免费的

**Reply by Minemetero** [reply]
我拿个树莓派 :)

**Reply by lingbopro** [reply]
emmm...

**Comment by lingbopro** [comment]
因过时而关闭lol

**Reply by Minemetero** [reply]
lol
